From edd16019335ba23e9c40c2711532861647de8fb1 Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Mon, 16 Oct 2017 12:25:19 +0530
Subject: [PATCH 046/767] arm64: dts: freescale: ls1012a: fix RGMII tx delay
 issue

Recently logic to enable RGMII tx delay was changed by
below patch.

https://patchwork.kernel.org/patch/9447581/

Based on the patch, enabling tx delay again using rgmii-txid.

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
Signed-off-by: Anjaneyulu Jagarlmudi <anji.jagarlmudi@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
index fb2b1323f07c..44f2f76cfd98 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
@@ -124,7 +124,7 @@
 		fsl,gemac-bus-id = < 0x1 >;	/* BUS_ID */
 		fsl,gemac-phy-id = < 0x1 >;	/* PHY_ID */
 		fsl,mdio-mux-val = <0x0>;
-		phy-mode = "rgmii";
+		phy-mode = "rgmii-txid";
 		fsl,pfe-phy-if-flags = <0x0>;
 
 		mdio@0 {
-- 
2.17.0

